token/program-2022/src/extension/confidential_transfer/ciphertext_extraction.rs

```diff
@@ -371,7 +371,7 @@ impl From<TransferWithFeeProofContext> for TransferWithFeeProofContextInfo {
 impl TransferWithFeeProofContextInfo {
     /// Create a transfer proof context information needed to process a [Transfer] instruction from
     /// split proof contexts after verifying their consistency.
-    pub fn new(
+    pub fn verify_and_generate(
```

Comment:
nit: I'm confused about the term  `generate`, which I see as more about creating proofs, whereas here you're more `create`ing or `extract`ing. How about `verify_and_extract`?

Reply:
Ah right, I like that much more. Thanks!